spec/commands/v2/publish_spec.rb

```diff
@@ -71,6 +71,25 @@
       end
     end
 
+    context "with major update_type sent in the payload" do
+      it "updates the temporary_published_at time to now" do
+        described_class.call(payload)
+
+        expect(draft_item.reload.temporary_published_at)
+          .to be_within(1.second).of(Time.zone.now)
+      end
```

Comment:
to save having to do the within(1.second) you could wrap these in Timecop.freeze blocks
eg: https://github.com/alphagov/publishing-api/blob/be9d4a2bfa70402196480f6573965ffac680fa7a/spec/queries/get_expanded_links_spec.rb#L66-L73

Or if you're doing loads you can do them in before and ends: https://github.com/alphagov/publishing-api/blob/faf9038a2a3ecef67e8b6f66d6539ee1f537c40a/spec/lib/tasks/taxonomy_event_log_spec.rb#L3-L10